casts/splitNavigator/src/SplitNavigator/createSplitNavigator.tsx

```diff
@@ -167,11 +169,12 @@ function UnfoldedSplitNavigator({
                             />
                         </View>
                         <View style={splitStyles.detail}>
-                            <ScreenContainer
+                            <MaybeScreenContainer
                                 // If not disabling the container for native, it will crash on iOS.
                                 // It happens due to an error in `react-native-reanimated`:
                                 // https://github.com/software-mansion/react-native-reanimated/issues/216
-                                enabled={!doesSupportNative}
+                                // enabled={!doesSupportNative}
+                                enabled
```

Comment:
Maybe to move this `enabled` inside of MaybeScreenContainer itself?
Somewhere here: https://github.com/tonlabs/UIKit/blob/05d75b1f93f4d6ea3345f82197a93ae3f7b599ff/casts/splitNavigator/src/SplitNavigator/ScreenFallback.tsx#L37

Reply:
actually `MaybeScreenContainer` is just copy-pasted from https://github.com/react-navigation/react-navigation/blob/main/packages/bottom-tabs/src/views/ScreenFallback.tsx, hence I don't want to make changes there :)